.travis.yml

```diff
@@ -1,12 +1,21 @@
 language: rust
+
 rust:
   - stable
   - beta
   - nightly
+
 os:
   - linux
 
 matrix:
   fast_finish: true
+
 notifications:
   email: false
+
+deploy:
```

Comment:
There shouldn't be a `deploy` section here, it should just be `script`. Here's some more information:

https://docs.travis-ci.com/user/customizing-the-build#Customizing-the-Build-Step